### PR TITLE
api changed, regenerate code again

### PR DESCRIPTION
**Pull request by chai2010**

### CODE REVIEW

File: pkg/cmd/api/spec/metadata.swagger.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "metadata.proto",
```
Comment (chilianyi) [inline pkg/cmd/api/spec/metadata.swagger.json]: What is this swagger for?

Comment (chai2010) [reply pkg/cmd/api/spec/metadata.swagger.json]: this generated swagger file should be ignored

Comment (chai2010) [reply pkg/cmd/api/spec/metadata.swagger.json]: i will fix it

Comment (chai2010) [reply pkg/cmd/api/spec/metadata.swagger.json]: done

File: pkg/service.pb/metadata.pb.go [hunk 2]
```diff
@@ -0,0 +1,844 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// source: metadata.proto
+
```
Comment (chilianyi) [inline pkg/service.pb/metadata.pb.go]: Here two should be ignore too.

Comment (chai2010) [reply pkg/service.pb/metadata.pb.go]: the metadata.pb.go will be used in unit test, for check config and db orm.

Comment (chai2010) [reply pkg/service.pb/metadata.pb.go]: the generated metadata.validator.pb.go is empty, it is harmless.
if we need ignore it, we must change the generate code command.

**Pull request merged**